### PR TITLE
Refine forking/PR details in “GitHub for complete beginners” doc

### LINKED ISSUE #6875: Issue with "GitHub for complete beginners": (confusing phrasing)
MDN URL: https://developer.mozilla.org/en-US/docs/MDN/Contribute/GitHub_beginners

#### What information was incorrect, unhelpful, or incomplete?

> This is required for you to submit change requests to the main version of the repo...you can't submit changes directly to the main version of the repo.

#### Specific section or headline?
Forking and cloning

#### What did you expect to see?
We can or we can't submit changes to the main repo? I'm a bit confused.

#### Did you test this? If so, how?
No

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/mdn/contribute/github_beginners`
* MDN URL: https://developer.mozilla.org/en-US/docs/MDN/Contribute/GitHub_beginners
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mdn/contribute/github_beginners/index.html
* Last commit: https://github.com/mdn/content/commit/fcc38b5d1b88f3d841165467c4d789a90f8721b6
* Document last modified: 2021-03-25T16:41:51.000Z

</details>

### PR DESCRIPTION
**Pull request by NegiAkash890**
## What was wrong/why is this fix needed? (quick summary only)

Modified statement to add clarity

## Issue number (if there is an associated issue)

Fixes #6875 

## URL

https://developer.mozilla.org/en-US/docs/MDN/Contribute/GitHub_beginners#forking_and_cloning

### CODE REVIEW

**Comment by teoli2003** [comment]
@ConsciousClay , can you check if this solve the issue you raised?

**Comment by ConsciousClay** [comment]
I don't know how to modify the first commit or to add another commit to this PR. Perhaps you can explain that to me? I'm using GitHub's UI for now. Until then, I'll write my amendments here:

> Due to security reasons, you can't submit changes _directly_ to the main version of the repo. To submit changes to the main repo, we first must make a **fork** of it, then we **push** our changes to our fork, and finally we create a **pull request** to push those changes from our fork to the main repo.

How's this?

**Comment by Ryuno-Ki** [comment]
> I don't know how to modify the first commit or to add another commit to this PR.

Go to https://github.com/mdn/content/pull/6895/files#diff-d5176d797ce0255ebc83adad717b21418946b5fee26c423bbc0a5c7740517c3a and find the three dots in the upper right corner. There, you will find an „Edit file” option, that will bring you to GitHub's Edit UI. Add more commits to this PR this way.

**Comment by ConsciousClay** [comment]
> Go to https://github.com/mdn/content/pull/6895/files#diff-d5176d797ce0255ebc83adad717b21418946b5fee26c423bbc0a5c7740517c3a and find the three dots in the upper right corner. There, you will find an „Edit file” option, that will bring you to GitHub's Edit UI. Add more commits to this PR this way.

The option to edit the file is grayed out. The original commit isn't mine. What can I do to make changes in this case?

**Review by bound-variable — COMMENTED** [review]
How about:

Due to security reasons, you can't submit changes directly to the main version of the repo. To submit changes to the main repo we must first make a fork of it, then we push our changes to our fork, and finally we create a pull request to push those changes from our fork to the main repo.

**Comment by Ryuno-Ki** [comment]
Oh, in that case, you will need to use git from a terminal.

I documented my process in another PR: https://github.com/mdn/content/pull/4109#issuecomment-858051849

**Comment by sideshowbarker** [comment]
It seems unclear what this PR is waiting on at this point.

**Comment by ConsciousClay** [comment]
I was told to stop making contributions.

File: files/en-us/mdn/contribute/github_beginners/index.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h3 id="forking_and_cloning">Forking and cloning</h3>
   <li>Cloning means making a local copy of a repo for you to work on (i.e. on your local hard drive).</li>
 </ul>
 
-<p>It is possible to do the two things separately, but in practice you will nearly always do them together when contributing to other people's projects. You should first make a fork of each repo you want to work on. This is required for you to submit change requests to the main version of the repo (we'll learn how to create a pull request later on). Due to security reasons, you can't submit changes directly to the main version of the repo.</p>
+<p>It is possible to do the two things separately, but in practice you will nearly always do them together when contributing to other people's projects. You should first make a fork of each repo you want to work on. This is required for you to submit change requests to the main version of the repo (we'll learn how to create a pull request later on). Due to security reasons, you can't submit changes directly to the main version of the repo. In order to do so we should first fork the main repo, then push the changes to our fork and then finally we can create a pull request to push changes from our fork to main repo.</p>
```
Comment (sideshowbarker) [inline files/en-us/mdn/contribute/github_beginners/index.html:81]:
```suggestion
<p>It is possible to do the two things separately, but in practice you will nearly always do them together when contributing to other people's projects. You should first make a fork of each repo you want to work on. This is required for you to submit change requests to the main version of the repo (we'll learn how to create a pull request later on). Due to security reasons, you can't submit changes directly to the main version of the repo. So to submit changes, first fork the main repo, then push the changes to your fork, and then create a pull request to get the changes from your fork merged into the main repo.</p>
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by Pidut** [comment]
- Nhjsbpow open I mega888 casino Malaysia and Asia Jason power source sound mic high quality - - - - there is **nk s**@********

**Pull request merged**